src/providers/twitch/api/Helix.hpp

```diff
@@ -616,6 +616,22 @@ enum class HelixStartCommercialError {
     Forwarded,
 };
 
+enum class HelixGetGlobalBadgesError {
+    Unknown,
+    UserNotAuthorized,
```

Comment:
These endpoints do not yield 403 (badge info is unscoped), so we don't need `UserNotAuthorized`. The same applies for the channel-version of the endpoint

Reply:
I set that error message to 401 when the token is invalid, maybe I should rename it 🤔

Reply:
Fixed, thank you for the correction 😁